lib/ferrum/browser.rb

```diff
@@ -77,6 +77,11 @@ def extensions
       end
     end
 
+    def evaluate_on_new_document(javascript = nil)
+      extensions << (block_given? ? yield : javascript)
+    end
+    alias evaluate_on_new_document= evaluate_on_new_document
```

Comment:
`evaluate_on_new_document=` confuses me as I would think that I can override previously passed script, let's leave only `evaluate_on_new_document`